docs/websockets.rst

```diff
@@ -361,12 +361,13 @@ Whenever a protocol cache is being updated for protocols like Curve, Convex, Gea
 
     {
         "data": {
+            "type":"protocol_cache_updates",
             "protocol":"curve",
             "chain":"ethereum",
             "processed": 324,
             "total":3042,
         },
-        "type":"protocol_cache_updates"
+        "type":"progress_updates"
```

Comment:
for all these changes you need to explain what data now holds. So you need to also edit the entry below. For every place you touch.